tests/security/MemberTest.yml

```diff
@@ -42,7 +42,7 @@ Member:
   test:
     FirstName: Test
     Surname: User
-    Email: sam@silverstripe.com
+    Email: dummy@silverstripe.tld
```

Comment:
Oh, ok. I thought .tld was "protected" from becoming a real TLD as well, but I guess that's not true and you never know with ICANN. These changes here are in line with the ones I made to the master branch though… is this something that should be changed in master as well?